Arithmetic.pp

```diff
@@ -44,12 +44,12 @@
 %token  bracket_  \(
 %token _bracket   \)
 %token  comma     ,
-%token  number    (0|[1-9]\d*)(\.\d+)?([eE][\+\-]?\d+)?
+%token  number    (0|[1-9]\d*)(\.\d+)?([eE][\+\-]?\d+)?\b
```

Comment:
You can move the `number` token after the `constant` token to avoid `7A` to be consumed as a number `7` and a constant `A` instead of the constant `7A`.